maps/utk.yml

```
@@ -3454,6 +3454,44 @@ properties:
     requirement: required
     sample_values:
     - http://rightsstatements.org/vocab/InC/1.0/
+  sequence:
+    available_on:
+            class:
+            - Attachment
+            - GenericWork
+            - Image
+            - Video
+            - Audio
+            - Pdf
+            - Book
+            - CompoundObject
+            - Newspaper
+    cardinality:
+            maximum: 1
+            minimum: 0
+    multi_value: true
+    controlled_values:
+            format: http://www.w3.org/2001/XMLSchema#integer
+            sources:
+            - 'null'
+    definition:
+           default: Numberic order in which to display pages or other items
+    display_label:
+            default: Sequence
+    index_documentation: displayable, stored_searchable
+    indexing:
+    - displayable
+    - stored_searchable
```

We have some linting things that are unhappy with this.  Let's just change here.

```suggestion
  sequence:
    available_on:
      class:
      - Attachment
      - GenericWork
      - Image
      - Video
      - Audio
      - Pdf
      - Book
      - CompoundObject
      - Newspaper
    cardinality:
      maximum: 1
      minimum: 0
    multi_value: true
    controlled_values:
    format: http://www.w3.org/2001/XMLSchema#integer
    sources:
    - 'null'
    definition:
      default: Numeric order in which to display pages or other items
    display_label:
      default: Sequence
    index_documentation: displayable, stored_searchable
    indexing:
    - displayable
    - stored_searchable
```